Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
+ */
 @interface ARTCrypto : NSObject
 
-+ (ARTCipherParams *)getDefaultParams:(NSDictionary *)values;
+/**
+ Retrieves, or optionally sets, the `ARTCipherParams` for the channel.
+ @param params Overrides the default parameters. A suitable key must be provided as a minimum.
+ @return `ARTCipherParams` A ARTCipherParams object, using the default values for any field not supplied.
```

Comment:
@maratal this question applies to all of the `@return` tags where you've written the return type as part of the tag – do you think this is necessary? The return type is already displayed in the Jazzy documentation as part of the method signature:

> <img width="727" alt="Screenshot 2022-07-05 at 17 42 22" src="https://user-images.githubusercontent.com/53756884/177413017-d3a80e90-01fa-448b-8dc7-0d8feff5d2c1.png">

If you look at Apple's documentation ([example](https://developer.apple.com/documentation/foundation/nsstring/1410710-lengthofbytes)), the "Return Value" section doesn't repeat the return type; rather, it's used to provide semantic information about the return value:

> <img width="755" alt="Screenshot 2022-07-05 at 17 44 50" src="https://user-images.githubusercontent.com/53756884/177413222-c217c020-6079-4101-bbfe-c71703684346.png">

Reply:
https://github.com/ably/ably-cocoa/pull/1439/commits/451185143a27f803a68b015fdd798df5e5e369f4